canonical-websites/www.ubuntu.com
# Support overview

## Done

Converted the /support/ page to VBT with new content added and some new patterns/bugfixes.
 - Temp fix to the padding of inline-images (Current pattern has double the amount proposed)
 - Added back the django template tag for `<body>` classes.
 - Added a `had-background` class which gives the suru site background (This should be back-ported to other pages that have already been completed)
 - Added an x-light strip variant now that the site background isn't white
 - Added a `.u-full-width` utility class to force an element to be 100% wide.
 - Fixed some issues merging various changes to inline-images (repaired the `--compact` version on /server/maas)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support](http://0.0.0.0:8001/support)
- Check the page in all viewports
 - Check the page against the [copy](https://docs.google.com/document/d/1liY2ulrI3JiICSJquTn9cH0Hu-R6H3ufmEmYhive-QI/edit#)
 - Check that the lists on [/server/maas](http://0.0.0.0:8001/support) still look good

## Issue / Card

Fixes #1590 
Fixes #1802 

## [Demo](http://www.ubuntu.com-support-overview.demo.haus/)
